modules/juce_opengl/opengl/juce_OpenGLGraphicsContext.cpp

```diff
@@ -114,7 +114,8 @@ struct CachedImageList  : public ReferenceCountedObject,
             if (textureNeedsReloading && pixelData != nullptr)
             {
                 textureNeedsReloading = false;
-                texture.loadImage (Image (*pixelData));
+                auto nativeColourSpaceImage = juce_returnImageWithNativeColourSpace (Image (*pixelData));
+                texture.loadImage (nativeColourSpaceImage);

```

Comment:
I think that using an intermediate variable might slow things down without an `std::move`.. C++..